chart/csi-vault/templates/csi-driver.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+    name: {{ .Values.driverName }}
+spec:
+    attachRequired: {{ .Values.csidriver.attachRequired }}
+    podInfoOnMount: {{ .Values.csidriver.podInfoOnMount }}
```

Comment:
I don't know if `.Values.csidriver.podInfoOnMount` should be a variable. CSI driver will not work without this, right?

Reply:
Yes, `podInfoOnMount` must be true. This value is set to true.